test/epiviewpoint_web/controllers/import_controller_test.exs

```diff
@@ -87,14 +87,68 @@ defmodule EpiViewpointWeb.ImportControllerTest do
       assert conn |> redirected_to() == "/import/start"
       assert "Invalid mm-dd-yyyy format: 06/02/bb" = Session.get_import_error_message(conn)
     end
+
+    test "accepts ndjson file upload", %{conn: conn} do
+      temp_file_path =
+        """
+        {"search_firstname_2":"Alice","search_lastname_1":"Testuser","dateofbirth_8":"01/01/1970","datecollected_36":"06/02/2020","resultdate_42":"06/01/2020","datereportedtolhd_44":"06/03/2020","result_39":"positive","glorp":"393","person_tid":"alice"}
```

Comment:
Should there be this blank line?

Reply:
not sure if I understand correctly, the `"""` will not create newline at the beginning, as opposed to `"`

<img width="164" alt="image" src="https://github.com/user-attachments/assets/13e64ebc-cd10-44de-94d1-10b9c5424f08">

Reply:
![image](https://github.com/user-attachments/assets/a22c3b48-cf5a-41fd-b243-6fac12c10141)

Line 88 looks like a blank line but maybe it's just how it's rendered?